stytch/b2b/organizations/types.go

```diff
@@ -187,6 +187,33 @@ type GetParams struct {
 	OrganizationID string `json:"organization_id,omitempty"`
 }
 
+type GithubProviderInfo struct {
+	ProviderSubject   string   `json:"provider_subject,omitempty"`
+	ProviderTenantIds []string `json:"provider_tenant_ids,omitempty"`
+	AccessToken       string   `json:"access_token,omitempty"`
+	Scopes            []string `json:"scopes,omitempty"`
+}
+
+// HubspOTPRoviderInfo:
+type HubspOTPRoviderInfo struct {
```

Comment:
hmm this is kind of weird...I wonder if it's trying to do an override to capitalize the substring `OTP`